support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -71,4 +72,28 @@ export const tests: Tests = {
     targetPage: allLandingPagesAndThankyouPages,
     seed: 12,
   },
+
+  usLandingPageZipCodeFieldTest: {
```

Comment:
We only actually want to run this on single contributions. We'll have to keep that in mind when analysing

Reply:
This is now moot as we're running the test across both contribution types.